tools/lib/h5tools.h

```diff
@@ -578,15 +578,15 @@ typedef struct h5tools_vfd_info_t {
     } u;
 } h5tools_vfd_info_t;
 
-/* This enum should match the entries in the above 'volnames'
- * since they are indices into the 'volnames' array. */
+/* This enum should match the entries in the 'volnames'
+ * array since they are indices into that array. */
 typedef enum {
     NATIVE_VOL_IDX = 0,
     PASS_THROUGH_VOL_IDX,
 } vol_idx;
```

Comment:
We should probably do the same this with the VOL array that we do with the VFD array.

Reply:
Agreed. Anything to make the maintainability there better.